src/main/java/marquez/api/resources/DatasourceResource.java

```diff
@@ -5,19 +5,25 @@
 import com.codahale.metrics.annotation.ExceptionMetered;
 import com.codahale.metrics.annotation.ResponseMetered;
 import com.codahale.metrics.annotation.Timed;
-import java.util.Collections;
+import java.util.List;
 import javax.ws.rs.DefaultValue;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.Response;
+import marquez.api.mappers.DatasourceResponseMapper;
 import marquez.api.models.DatasourcesResponse;
+import marquez.service.DatasourceService;
 
 @Path("/api/v1")
 public final class DatasourceResource {
 
-  public DatasourceResource() {}
+  private final DatasourceService datasourceService;
+
+  public DatasourceResource(DatasourceService dataSourceService) {
```

Comment:
minor: by convention, we'll want to define the constructor as:

```java
public DatasourceResource(final DatasourceService datasourceService) {
  ...
}
```
That is,
*  Args passed to the construct are always `final`
* `dataSourceService` **=>** `datasourceService`

Reply:
Will be fixed in the next revision.

Reply:
Please see PRs #394, #387, and #384, where this is fixed.